src/test/java/com/commercetools/sync/states/utils/StateReferenceResolutionUtilsTest.java

```diff
@@ -6,23 +6,26 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-import io.sphere.sdk.expansion.ExpansionPath;
 import io.sphere.sdk.models.Reference;
 import io.sphere.sdk.states.State;
 import io.sphere.sdk.states.StateDraft;
 import io.sphere.sdk.states.queries.StateQuery;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.UUID;
 import javax.annotation.Nonnull;
 import org.junit.jupiter.api.Test;
 
 class StateReferenceResolutionUtilsTest {
 
+  Map<String, String> idToKeyValueMap = new HashMap<>();
```

Comment:
Need clean up?

Reply:
I am creating Id's using UUID generator for every test and also Cache is not cleared manually anywhere in the code. May be it is good to not clean up cache. WDYT?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/701/commits/6fde9c6afded1a6a8fae4d46d300090d04bc8111